src/models/NumberModel.ts

```diff
@@ -17,3 +17,8 @@ export interface SystemNumber {
     systemNumber: string;
     testNumberKey: number;
 }
+
+export interface PlateSerialNumber {
+    plateSerialNumber: string;
+    testNumberKey: number;
```

Comment:
suggest to make a base `interface NumberKey { testNumberKey: number }` all others extend and add their specific attributes.

Reply:
done